Source/Core/Core/NetPlayClient.cpp

```diff
@@ -1534,6 +1566,30 @@ bool NetPlayClient::WiimoteUpdate(int _number, u8* data, const u8 size, u8 repor
   return true;
 }
 
+void NetPlayClient::HostPadPoll()
+{
+  if (m_local_player->pid == 1)
+  {
+    for (PadMapping map = 0; map < 4; map++)
```

Comment:
`map < m_pad_map.size()`

Reply:
I don't see any reason to do it like this when it isn't anywhere else. `4` is a hardware constant in this case.

Reply:
Additionally, this produces a warning about comparing integers of different signedness.

Reply:
> I don't see any reason to do it like this when it isn't anywhere else. 

That's arguably an issue with the other code then

> 4 is a hardware constant in this case.

OK, but you're iterating and indexing array of the same size, there's no need to hardcode the upper bound.

> Additionally, this produces a warning about comparing integers of different signedness.

Then just use `size_t`? That's a trivially amendable issue.

Reply:
Fair enough, I suppose I can just cast it to `PadMapping` as needed. Also I checked and there's only a few cases of the magic constant anyways.